src/plugins/mer/merbuildengineoptionswidget.cpp

```diff
@@ -731,8 +731,14 @@ void MerBuildEngineOptionsWidget::onSshTimeoutChanged(int timeout)
 
 void MerBuildEngineOptionsWidget::onSshPortChanged(quint16 port)
 {
+    BuildEngine *const buildEngine = m_buildEngines[m_virtualMachine];
+
     //store keys to be saved on save click
-    m_sshPort[m_buildEngines[m_virtualMachine]] = port;
+    m_sshPort[buildEngine] = port;
+
+    m_ui->buildEngineDetailsWidget->setSshPortOccupied(
+                    (buildEngine->virtualMachine()->isOff() || port != buildEngine->sshPort())
+                    && Sfdk::isPortOccupied(port));
```

Comment:
Bad indent, should be -2 levels.

Reply:
My bad, ClangFormat plugin was disabled on my QtC (maybe during update). However, I noticed that the provided settings (.clang-format file in repo) is not always produce correct indents. So, I have to correct indents myself in some cases.